website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+                    <div data-bind='if:page() === REMOVE_NO_CHILDREN'>
+                        <div class="form-group" data-bind="if:contributorToRemove">
+                            <span>Remove <b>{{contributorToRemove()["fullname"]}}</b> from {{title}}"?</span>
+                        </div>
+
+                    </div><!-- end removeNoChildren page -->
+
+                    <!-- removeAll page -->
+                    <div data-bind='if:page() === REMOVE_ALL'>
+                        <div data-bind="visible:titlesToRemove().length">
+                            <div class="panel panel-default">
+                                <div class="panel-body">
+                                    <div class="form-group" data-bind="if:contributorToRemove">
+                                        <span><b>{{contributorToRemove()["fullname"]}} will be</b> removed from the following projects and/or components.</span>
+                                    </div>
+                                    <div class="col-md-8" align="left">
+                                        <ul data-bind="foreach: { data: titlesToRemove(), as: 'item' }">
+                                            <li>
+                                                <h4 class="f-w-lg" data-bind="text: item"></h4>
+                                            </li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div data-bind="visible:titlesToKeep().length">
+                            <div class="panel panel-default">
+                                <div class="panel-body">
+                                    <div class="form-group" data-bind="if:contributorToRemove">
+                                        <span><b>{{contributorToRemove()["fullname"]}} cannot</b> be removed from the following projects and/or components.</span>
+                                    </div>
+                                    <div class="col-md-8" align="left">
+                                        <ul data-bind="foreach: { data: titlesToKeep(), as: 'item' }">
+                                            <li>
+                                                <h4 class="f-w-lg" data-bind="text: item"></h4>
+                                            </li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div><!-- end removeAll page -->
+                </div>
+                <div data-bind="if: !canRemoveNode() && !pageChanged()">
+                    <span><b>{{contributorToRemove()["fullname"]}}</b> cannot be removed as a contributor.  You need at least one administrator, bibliographic contributor, and one registered user.</span>
```

Comment:
The grammar of the second sentence seems off, because the parallelism of "one" is only partially applied. Maybe one of the following?

> You need at least one administrator, bibliographic contributor, and registered user.

or

> You need at least one administrator, one bibliographic contributor, and one registered user.

Or maybe I don't understand the nuance?